### PR TITLE
Add recipe for perl-logger-simple

### PR DESCRIPTION
**Pull request by npavlovikj**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by npavlovikj** [comment]
Missing dependencies are in bioconda...

File: recipes/perl-logger-simple/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "perl-logger-simple" %}
+{% set version = "2.0" %}
+{% set sha256 = "2e63fd3508775b5902132ba1bfb03b42bee468dfaf35dfe42e1909ff6d291b2d" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/T/TS/TSTANLEY/Logger-Simple-2.0.tar.gz
```
Comment (chrisburr) [inline recipes/perl-logger-simple/meta.yaml]:
```suggestion
  url: https://cpan.metacpan.org/authors/id/T/TS/TSTANLEY/Logger-Simple-{{ version }}.tar.gz
```

**Comment by chrisburr** [comment]
Can you make a pull request to migrate the dependencies over to conda-forge?

**Comment by npavlovikj** [comment]
I am trying Chris, I just need to find the basic, no dependencies perl recipes first to transfer :)

**Pull request closed without merging**